data/map.txt

```diff
@@ -32288,6 +32297,7 @@ wormhole "Wormhole Link"
 	mappable
 	link Statina Vaticanus
 	link Vaticanus Statina
+	color 0.5 0.1 0.3
```

Comment:
mhm.... NO DATA FILE CHANGES!!!

Reply:
Yea, but you do not change the code so it is a neccessity to make the data file changes.
-> separated PR